### PR TITLE
#VEBT-1992 added feature toggle for form 1995

### PR DESCRIPTION
**Pull request by wafimohamed**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**: Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
- _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

## Summary

- _(Summarize the changes that have been made to the platform)_
- replaced environment.isProduction() to toggle in form 1995 so we can easily just switch it to on when we want to show it in prod.
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_ VEBT
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_ N/A

## Related issue(s)
Description
With the feature flag on it we will have an off/on switch that will allow us to turn it "on" in prod on the 6th with out another deployment. Until then the changes still stay in staging.

- _Link to ticket created in va.gov-team repo_ https://jira.devops.va.gov/browse/VEBT-311
department-of-veterans-affairs/va.gov-team#0000
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#0000
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by GovNapoleon — DISMISSED** [review]

File: src/applications/edu-benefits/1995/helpers.jsx [hunk 1]
```diff
@@ -1,15 +1,13 @@
 import React from 'react';
-import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+// import environment from '@department-of-veterans-affairs/platform-utilities/environment';
```
Comment (micahchiang) [inline src/applications/edu-benefits/1995/helpers.jsx:2]: Can this comment be removed?

File: src/applications/edu-benefits/hooks/useSetToggleParam.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+
+export const useSetToggleParam = toggleValue => {
+  useEffect(
+    () => {
+      const urlParams = new URLSearchParams(window.location.search);
+      if (
+        toggleValue !== undefined &&
+        toggleValue !== null &&
+        toggleValue !== 'undefined'
```
Comment (micahchiang) [inline src/applications/edu-benefits/hooks/useSetToggleParam.js:10]: Can this check just be !toggleValue?

Comment (wafimohamed) [reply src/applications/edu-benefits/hooks/useSetToggleParam.js:10]: yeah sure will change that

Comment (wafimohamed) [reply src/applications/edu-benefits/hooks/useSetToggleParam.js:10]: actually that cause kinda of infinite loop .page keeps loading

File: src/applications/edu-benefits/1995/helpers.jsx (hunk 1 above)
Comment (wafimohamed) [reply src/applications/edu-benefits/1995/helpers.jsx:2]: sure

**Review by micahchiang — APPROVED** [review]

**Review by jsimonVA — APPROVED** [review]
LGTM

**Pull request merged**